appcelerator/titanium_mobile
# fix(android): crash with old "ti.playservices" in 9.0.0

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-27747

**Summary:**
- Fixed regression caused by PR #11449
- Calling old "ti.playservices" module's `makeGooglePlayServicesAvailable()` method would cause a crash in 9.0.0. Re-added support for legacy hybrid modules.

**Test:**
1. Follow the test case written in [TIMOB-27747](https://jira.appcelerator.org/browse/TIMOB-27747).
2. Verify app does not crash on app startup.
3. Verify dialog shows "makeGooglePlayServicesAvailable" info.
